src/main/java/org/isf/disease/dto/DiseaseDTO.java

```diff
@@ -32,32 +32,32 @@
 public class DiseaseDTO {
 
 	@NotNull
-	@Schema(description = "Disease code", example = "99")
+	@Schema(description = "Disease code", example = "99", maxLength = 10)
 	private String code;
 
 	@NotNull
-	@Schema(description = "Disease description")
+	@Schema(description = "Disease description", maxLength = 50)
     private String description;
```

Comment:
Here is the that looks like the indention is wrong;   there are other occurrences too that I haven't bothered to mark.

Reply:
please how many tab space do you use in you IDE please.